### ISSUE TITLE
Build failure with invalid options for ln

### ISSUE THREAD

**Issue by Earnestly** [ytomino/headmaster#3]
Quick note, I'm getting a build failure with `make -C main` with the following error:

```
ln -sfh ../source source
ln: invalid option -- 'h'
Try 'ln --help' for more information.
```

This appears to originate from these commands:
```make
$(BINLN):
	ln -sfh $(BUILDDIR) $@

$(SOURCELN):
	ln -sfh $(SOURCEDIR) $@
```

Note, `-h` is not available in either GNU ln or POSIX ln so I'm not sure what it's supposed to do.

**Comment by ytomino** [comment]
`ln -h` is the function of BSD (including Darwin) ln, to follow no symbolic links.
I use this option to avoid to make nested links like `bin/bin/bin`.
Sorry, it's certainly unnecessary for this case.

**Comment by ytomino** [comment]
The fix is committed as 14eb7c07.
Thank you.

**Comment by Earnestly** [comment]
Thanks
